src/common/logger/get-system-info.ts

```diff
@@ -0,0 +1,23 @@
+import OS from "os";
```

Comment:
We might move this into another folder, since this info may not be used only in the logger